lib/rules/valid-v-bind-sync.js

```diff
@@ -82,6 +82,20 @@ function maybeNullObjectMemberExpression(node) {
   return false
 }
 
+function isValidIs(node) {
+  const { attributes } = node
+  const isAttribute = attributes.filter((attr) => {
+    if (attr.type === 'VAttribute' && attr.key.name === 'is') return attr
+  })
+  if (
+    isAttribute.length > 0 &&
+    ['tr', 'li', 'option'].includes(node.parent.name)
```

Comment:
Could you remove the element name check?
The `is` attribute can be used on all elements. There is no need to check the element name.